.flake8

```diff
@@ -1,3 +1,3 @@
 [flake8]
 max-line-length=120
-exclude = .git,__pycache__,dist,.venv/*,node_modules/*,utils/*,tests/.venv/*
+exclude = .git,__pycache__,dist,.venv/*,node_modules/*,utils/*,tests/.venv/*,proxies/live/apiproxy/resources/py/DecodeBase64TargetIdentifier.py
```

Comment:
Because lint does not understand the flow variable in the proxies javascripts and fails the build.
This took me ages to figure out until I found that the same is being done in the booking-and-referral-fhir-api that also decodes the target identifier

Reply:
Another reason to move away from python. See other comment about tech debt.